docker-compose.override.yaml

```diff
@@ -3,12 +3,18 @@ version: '3'
 services:
   mainnetwork:
     ports:
-      - "80:8000"
+      - "80:80"
+      - "8080:8080" # for webpack HMR
   web:
     build:
       context: .
       dockerfile: Dockerfile-dev
+  nginx:
```

Comment:
In Lumen, we use the same WebPack dev server as a proxy to the backend, so that we do not need an nginx server running.

Given that Nginx is quite light, probably it is ok to leave it as it is.